### ISSUE TITLE
Uses for the Sorter

### ISSUE THREAD

**Issue by moshemo** [reduxframework/redux-framework#2999]
It seems that the sorter can only be used for enabled, disabled and backup.  I am looking to be able to sort a list into different columns, but with the ability to determine the number of columns as well as the name and functionality of those columns.  

For instance, let's imagine I have a list of pages.  I want to then sort those pages into different columns to do different functions (whose names and functions I define).

Is that possible with the sorter?  

If not, any recommendations on how to do that?

Thanks.

**Comment by kprovance** [comment]
You can add as many sorter columns as you like, by adding them as an array under the options argument, including the default pages within.  Use the 'data' argument to list whatever (or all) pages you like (or, write your own function and feed that array into whichever column you like.

https://docs.reduxframework.com/core/fields/sorter/

**Comment by moshemo** [comment]
Okay -- perhaps I'm missing something, but I can't seem to figure this out.  

I just did a fresh (local) install of Wordpress and the only thing I have done so far is activated the redux framework plugin, activated the sample config file and made a slight change to the options section of the sorter type (the one with the id of opt-homepage-layout).

Here is the options I created:

                 'options'  => array(
                    'New Name'  => array(
                        'highlights' => 'Highlights',
                        'slider'     => 'Slider',
                        'staticpage' => 'Static Page',
                        'services'   => 'Services',
                        'tree'       => 'Apple'
                    ),
                    'disabled' => array(),
                    'backup'   => array('one' => 'One', 'two' => 'Two'),
                    'testing' => array()
                ),

And here is what it looks like:

![sorter-redux-framework](https://cloud.githubusercontent.com/assets/13052314/15478786/24d509c4-2125-11e6-9969-a401890feb6e.PNG)


Needless to say, this is not at all what I expected nor does it seem to conform to the code.  Am I doing something wrong?  If so, can you please tell me what I should do.  

What I would like is the ability to change the names of Enabled/Disabled, etc and to add new columns if I want to.

In short, what I really am looking for is the jquery ui droppable functionality.  

Thanks.

**Comment by kprovance** [comment]
I'm not able to try this based on the given code.  It;s incomplete.  If you want me to try it, I need to see the entire field araray.  In fact, if you could review this: https://github.com/reduxframework/redux-framework/blob/master/CONTRIBUTING.md and provide us with a support URL, that what's we need.

**Comment by kprovance** [comment]
You know, it just occurred to me, I get what those new columns are not appearing.  I remember this for some years back.  Every time you change something in the option argument, that is, add or remove a column, have to to reset the section to see the new changes.  Nature of the beast, unfortunately.

**Comment by moshemo** [comment]
I'm not sure what you mean by reset the section.  How do I do that?

**Comment by kprovance** [comment]
You click the "Reset Section" button at the top or bottom of the option panel?

**Comment by moshemo** [comment]
That did it :).  

Thanks.  Much appreciated.

**Comment by moshemo** [comment]
Can I suggest that you add that point (i.e., that you need to reset the section) into your documentation for the sorter (and any other types that it is relevant for).  It would save both all of us some time (you, because you won't have to deal with questions like this and us because we'll know how to get it to work properly right away).

**Comment by kprovance** [comment]
You can.  Can;t promise when I'll get around to it, since we do this stuff basically for free.

**Comment by moshemo** [comment]
I'd be happy to try and write the changes myself if that is possible.

**Comment by kprovance** [comment]
Access to edit our docs site isn;t just given away.  Like I said, I'll get to it.  :)
